config/detekt/detekt-override.yml

```diff
@@ -171,6 +171,7 @@ style:
       - '2'
       - '200'
       - '201'
+      - '400'
```

Comment:
バリデーションエラー系のHTTPステータスコードを400としました

[REST response code for invalid data](https://stackoverflow.com/questions/6123425/rest-response-code-for-invalid-data)

> What response code should be passed to client in case of following scenarios?
以下のシナリオの場合、どのようなレスポンスコードがクライアントに渡されるべきですか？

> Invalid data passed while user registration like wrong email format
> User name/ Email is already exists
I chose 403. I also found following that I feel can be used.

> 400 is the best choice in both cases. If you want to further clarify the error you can either change the Reason Phrase or include a body to explain the error.
どちらの場合も400が最適です。さらにエラーを明確にしたい場合は、Reason Phrase を変更するか、 エラーを説明するボディを含めることができます。 

> Invalid data passed while user registration like wrong email format
User name/ Email is already exists
I chose 403. I also found following that I feel can be used.
ユーザー登録時に渡されたデータが無効である（電子メールの形式が間違っているなど
ユーザー名/メールアドレスがすでに存在する
私は403を選択しました。また、以下のようなものもありましたので、使用できそうです。



----

一番票をもらったAnswer

> 412 - Precondition failed is used for conditional requests when using last-modified date and ETags.
412 - Precondition failed は、last-modified date と ETags を使用した条件付きリクエストに使用されます。

> 403 - Forbidden is used when the server wishes to prevent access to a resource.
403 - Forbidden は、サーバーがリソースへのアクセスを禁止したい場合に使用されます。

> The only other choice that is possible is 422 - Unprocessable entity.
他に可能な選択肢は、422 - Unprocessable Entity だけです。